Examples/Io/Root/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ target_link_libraries(
         Threads::Threads
     PRIVATE ROOT::Core ROOT::Hist ROOT::Tree
 )
+target_compile_definitions(ActsExamplesIoRoot PUBLIC ACTS_EXAMPLES_WITH_ROOT)
```

Comment:
Do we need this? I don't see where it's used, and usually public defines like this can be a bit painful.